### PR TITLE
Feature: clone home page demo

### PR DESCRIPTION
**Pull request by danglan910d1**
- ticket-id: https://phongdang7071.atlassian.net/browse/TRANNING-15
- description: create home page

### CODE REVIEW

File: assets/css/base.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    --border-color: #dbdbdb;
+
+    --header-height: 120px;
+    --navbar-height: 34px;
+    --header-with-search-height: calc(var(--header-height) - var(--navbar-height));
+    --header-sort-bar-height: 46px;
+}
+
+*{
+   box-sizing: inherit;
```
Comment (phongdang707) [inline assets/css/base.css]:
```suggestion
```
I think you should remove it

File: assets/css/base.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    text-decoration: none;
+}
+
+*, ::before, ::after {
+    box-sizing: border-box;
+}
+
+*, ::before, ::after {
+    box-sizing: border-box;
+}
```
Comment (phongdang707) [inline assets/css/base.css]: fix end of line

File: tiki.html [hunk 3]
```diff
@@ -0,0 +1,513 @@
...
+                                            </a>
+                                        </li>
+                                        <li class="header__search-history-item">
+                                            <a href="" class="item">
+                                                <img src="./assets/img/icon/searchicon1.png" alt="" class="header__search-btn-icon seacrh-icon">Iphone 13
+                                            </a>
+                                        </li>
+                                    </ul>
+
+                                    <div class="border-top" style="border-top: 1px solid rgb(225, 225, 225); margin-top: 6px;"></div>
```
Comment (phongdang707) [inline tiki.html]: Move style to css file

File: tiki.html [hunk 4]
```diff
@@ -0,0 +1,513 @@
...
+
+                                    <div class="bottom-widgets">
+                                        <div class="hot-keyword">
+                                            <h3 class="header__search-history-heading header__hot-keyword-heading" style="display: flex;">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 5]
```diff
@@ -0,0 +1,513 @@
...
+                                                <li class="header__search-history-item trending-product">
+                                                    <a href="" class="item">
+                                                        <img src="./assets/img/topsearchpro/hotpro6.jpg" alt="" class="thumb">
+                                                        <span class="trending-title">Mỹ phẩm</span>
+                                                    </a>
+                                                </li>
+                                            </ul>
+                                        </div>
+
+                                        <div class="border-top" style="border-top: 1px solid rgb(225, 225, 225); margin-top: 8px;"></div>
```
Comment (phongdang707) [inline tiki.html]: As above. U should add css and create style for this

File: tiki.html [hunk 6]
```diff
@@ -0,0 +1,513 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="Bottom__Root">
+                    <div class="root-container">
+                        <a href="#">
+                            <img src="./assets/img/logo/logotiki1.png" alt="" style="height: 1.8rem; width: 12.9rem;">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 7]
```diff
@@ -0,0 +1,513 @@
...
+                            <a href="" class="quicklink-container--item">Rau củ quả</a>
+                            <a href="" class="quicklink-container--item">Sữa, bơ, phô mai</a>
+                            <a href="" class="quicklink-container--item">Hải sản</a>
+                            <a href="" class="quicklink-container--item">Gạo, mì ăn liền</a>
+                            <a href="" class="quicklink-container--item">Đồ uống, bia, rượu</a>
+                            <a href="" class="quicklink-container--item">Bánh kẹo</a>
+                        </div>
+
+                        <div class="seller">
+                            <a href="" class="tiki-seller">
```
Comment (phongdang707) [inline tiki.html]: If href doesn't exist, u should add blank

<a href="#" class="tiki-seller">

File: tiki.html [hunk 8]
```diff
@@ -0,0 +1,513 @@
...
+        </div>
+
+        <div class="app__container">
+            <!-- Category Top -->
+            <div class="category-top">
+                <div class="container__category-top">
+                    <div class="category-top__style">
+                        <div class="content__category-top">
+                            <div class="cateslide">
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 9]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Thịt, Rau củ
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 10]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Bách Bóa
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 11]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Nhà Cửa
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 12]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Điện Tử
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 13]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Thiết Bị Số
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 14]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Điện Thoại
+                                    </div>
+                                </a>
+                                <a href="">
+                                    <div class="catename">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 15]
```diff
@@ -0,0 +1,513 @@
...
+                                        Mẹ & Bé
+                                    </div>
+                                </a>
+                                <a href="">
+                                    <div class="catename">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 16]
```diff
@@ -0,0 +1,513 @@
...
+                                        Gia Dụng
+                                    </div>
+                                </a>
+                                <a href="">
+                                    <div class="catename">
+                                        Thời Trang Nữ
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 17]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Thời Trang Nam
+                                    </div>
+                                </a>
+                                <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 18]
```diff
@@ -0,0 +1,513 @@
...
+                                    <div class="catename">
+                                        Giày Nữ
+                                    </div>
+                                </a>
+                                <a href="">
+                                    <div class="catename">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 19]
```diff
@@ -0,0 +1,513 @@
...
+                    </div>
+                </div>
+            </div>
+
+            <!-- Home Banner Top -->
+            <div class="grid wide">
+                <div class="banner-top">
+                    <div class="container__banner-top">
+                        <div class="banner-top__right">
+                            <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 20]
```diff
@@ -0,0 +1,513 @@
...
+                                <img src="./assets/img/bannertop/right1.webp" alt="">
+                            </a>
+
+                            <a href="" class="back-left">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 21]
```diff
@@ -0,0 +1,513 @@
...
+                                <img src="./assets/img/icon/backbanner.png" alt="">
+                            </a>
+
+                            <a href="" class="back-right">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 22]
```diff
@@ -0,0 +1,513 @@
...
+                                <img src="./assets/img/icon/backbanner.png" alt="">
+                            </a>
+                        </div>
+
+                        <div class="banner-top__left">
+                            <a href="">
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 23]
```diff
@@ -0,0 +1,513 @@
...
+                                        
+                                        <div class="deal-list">
+                                            <div class="deal-pro">
+                                                <div class="deal-pro__item">
+                                                    <div class="deal-pro__image">
+                                                        <img src="./assets/img/dealpro/dealpro1.webp" alt="">
+                                                    </div>
+                                                    <div class="deal-price">
+                                                        <span  class="price">99.000 đ</span>
+                                                        <span  class="price-discount">-80%</span>
```
Comment (phongdang707) [inline tiki.html]: remove space

File: tiki.html [hunk 24]
```diff
@@ -0,0 +1,513 @@
...
+                                            </div>
+
+                                            <div class="deal-pro">
+                                                <div class="deal-pro__item">
+                                                    <div class="deal-pro__image">
+                                                        <img src="./assets/img/dealpro/dealpro1.webp" alt="">
+                                                    </div>
+                                                    <div class="deal-price">
+                                                        <span  class="price">99.000 đ</span>
+                                                        <span  class="price-discount">-80%</span>
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 25]
```diff
@@ -0,0 +1,513 @@
...
+                                                    </div>
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 26]
```diff
@@ -0,0 +1,513 @@
...
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 27]
```diff
@@ -0,0 +1,513 @@
...
+                                            </div>
+
+                                            <div class="deal-pro">
+                                                <div class="deal-pro__item">
+                                                    <div class="deal-pro__image">
+                                                        <img src="./assets/img/dealpro/dealpro1.webp" alt="">
+                                                    </div>
+                                                    <div class="deal-price">
+                                                        <span  class="price">99.000 đ</span>
+                                                        <span  class="price-discount">-80%</span>
```
Comment (phongdang707) [inline tiki.html]: As above

File: tiki.html [hunk 28]
```diff
@@ -0,0 +1,513 @@
...
+                </div>
+            </div>
+        </div>
+
+        <footer class="footer">
+
+        </footer>
+    </div>
+</body>
+</html>
```
Comment (phongdang707) [inline tiki.html:508]: fix end of line

File: tiki.html [hunk 29]
```diff
@@ -0,0 +1,513 @@
...
+                                                        <span>Sắp hết bám</span>
+                                                        <img src="" alt="">
+                                                    </div>
+                                                </div>                                                
+                                            </div>
+                                            
+                                            
+                                        </div>
+
+                                        <a href="" class="deal-back__right">
```
Comment (phongdang707) [inline tiki.html]: As above

**Review by phongdang707 — COMMENTED** [review]
Change file tiki.html to index.html

File: assets/css/grid.css [hunk 30]
```diff
@@ -0,0 +1,591 @@
...
+    }
+
+    .wide .l-o-10 {
+        margin-left: 83.33333%;
+    }
+
+    .wide .l-o-11 {
+        margin-left: 91.66667%;
+    }
+}
```
Comment (phongdang707) [inline assets/css/grid.css:591]: please fix end of line

File: assets/css/tiki.css [hunk 31]
```diff
@@ -0,0 +1,720 @@
...
+/* .header__search-history-item a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 1.2rem;
+    line-height: 32px;
+    font-weight: 600;
+    color: var(--text-color) !important;
+} */
```
Comment (phongdang707) [inline assets/css/tiki.css]: remove this

File: assets/css/tiki.css [hunk 32]
```diff
@@ -0,0 +1,720 @@
...
+    position: absolute;
+    top: 0px;
+    left: 0px;
+}
+
+
+
+
+
+
```
Comment (phongdang707) [inline assets/css/tiki.css]: Why add multiple rows?

File: assets/css/base.css (hunk 2 above)
Comment (danglan910d1) [reply assets/css/base.css]: I don't understand about this issue
Can you help me? Please

File: assets/css/tiki.css (hunk 31 above)
Comment (danglan910d1) [reply assets/css/tiki.css]: Yes, i will

File: assets/css/tiki.css (hunk 32 above)
Comment (danglan910d1) [reply assets/css/tiki.css]: Sorry, I didn't notice this. I'll fix it

File: tiki.html (hunk 7 above)
Comment (danglan910d1) [reply tiki.html]: What do you think if I add #?

File: tiki.html (hunk 5 above)
Comment (danglan910d1) [reply tiki.html]: Yes, i will

**Review by phongdang707 — APPROVED** [review]

**Pull request merged**